web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte

```diff
@@ -190,12 +190,14 @@
     {/if}
 
     {#if isOwner}
-      <CircleIconButton
-        color="opaque"
-        icon={mdiDeleteOutline}
-        on:click={() => dispatch('delete')}
-        title={$t('delete')}
-      />
+      {#if !asset.isTrashed}
```

Comment:
I would still expect a delete button for a trashed asset to permanently delete it

Reply:
Yes, that's a good idea. I updated the code.